docs/translations/translations.json

```diff
@@ -1,98 +1,99 @@
 {
-  "homeQuickWord": "A quick word from our sponsors:",
-  "helpToTranslate": "Help to translate",
-  "editWebsiteColors": "Edit website colors",
-  "useDarkTheme": "Use dark theme",
-  "toggleTheme": "Toggle light/dark theme",
-  "toggleRTL": "Toggle right-to-left/left-to-right",
-  "toggleNotifications": "Toggle notifications panel",
-  "github": "GitHub repository",
-  "strapline": "React components for faster and easier web development. Build your own design system, or start with Material Design.",
-  "getStarted": "Get Started",
-  "installation": "Installation",
-  "installDescr": "Install Material-UI's source files via npm. We take care of injecting the CSS needed.",
-  "cdn": "or use a CDN.",
-  "loadFont": "Load the default Roboto font.",
-  "installButton": "Read installation docs",
-  "usage": "Usage",
-  "usageDescr": "Material-UI components work without any additional setup, and don't pollute the global scope.",
-  "usageButton": "Explore the docs",
-  "themes": "Premium themes",
-  "themesDescr": "Take your project to the next level with premium themes from our store – all built on Material-UI.",
-  "themesButton": "Browse themes",
-  "whosUsing": "Who's using Material-UI?",
-  "joinThese": "Join these and other great organisations!",
-  "usingMui": "Are you using Material-UI?",
-  "letUsKnow": "Let us know!",
-  "praise": "Praise for Material-UI",
-  "praiseDescr": "Here's what some of our users are saying.",
-  "footerCommunity": "Community",
-  "footerResources": "Resources",
-  "footerCompany": "Company",
-  "homeFooterRelease": "Currently {{versionNumber}}. Released under the {{license}}.",
-  "license": "MIT License",
-  "likeMui": "Help us keep running",
   "adblock": "If you don't mind tech related ads (no tracking or remarketing), and want to keep us running, please whitelist Material-UI in your blocker.",
-  "thanks": "Thank you!",
-  "editPage": "Edit this page",
-  "tableOfContents": "Contents",
-  "sourceCode": "Source code",
-  "traffic": "Traffic",
-  "newest": "Newest",
-  "stars": "GitHub stars",
-  "visit": "Visit the website",
-  "dashboardTitle": "Dashboard",
-  "dashboardDescr": "A minimal dashboard with taskbar and mini variant draw. The chart is courtesy of Recharts, but it is simple to substitute an alternative.",
-  "signInTitle": "Sign In",
-  "signInDescr": "A simple Sign In page.",
-  "signInSideTitle": "Sign-in side",
-  "signInSideDescr": "A simple Sign In side page.",
-  "signUpTitle": "Sign Up",
-  "signUpDescr": "A simple Sign Up page.",
-  "blogTitle": "Blog",
+  "albumDescr": "A responsive album / gallery page layout with a hero unit and footer.",
```

Comment:
can we avoid these in the future please? needless git blame indirection. otherwise add a lint rule to prevent regression.

Reply:
I have sorted asc. I was wondering if they were some sort of prefixing structure, as I was about to add a new key. Turns out, there is none.

Reply:
Not saying that sorting isn't useful. But it's easily forgotten and once every blue moon someone decides to update it :laughing:

Reply:
Agree, it's rarely kept up to date.